packages/utils/src/index.ts

```diff
@@ -9,3 +9,4 @@ export {
 export { calcDuration, formatBytes } from './lib/collect/implementation/utils';
 export { reportToStdout } from './lib/collect/implementation/report-to-stdout';
 export { reportToMd } from './lib/collect/implementation/report-to-md';
+export { importModule } from './lib/collect/implementation/load-file';
```

Comment:
Totally fine with this not being addressed in this PR to reduce scope, but I believe `utils` shoudn't have a `collect/implementation` folder in the end. That folder structure makes sense within `core`, but `utils` shouldn't really be aware of CLI commands like `collect`. The function themselves (`formatBytes`, `reportToMd`, `importModule`, etc.) should remain in `utils`, though, as they're pretty generic.

Reply:
it was a left over